UI/src/main-vue.js

```diff
@@ -45,7 +45,11 @@ if (document.getElementById("main")) {
                     document.body.setAttribute("data-lsmb-done", "true");
                 });
             });
-            window.__lsmbLoadLink = (url) => this.$router.push(url);
+            window.__lsmbLoadLink = (url) =>
+                this.$router.push(
+                    // eslint-disable-next-line no-useless-escape
+                    url.replace(/^https?:\/\/(?:[^@\/]+)/, "")
```

Comment:
Why do you exclude the @-sign? It can be used to separate username and password from the hostname, but if it does, the regex doesn't match. (No idea if this makes sense in the context ofLedgerSMB, actually)

Reply:
I didn't think it made sense for LedgerSMB.